rbac/rbac/middleware.py

```diff
@@ -152,10 +151,11 @@ def _create_user(username, tenant, request):
         return new_user
 
     @staticmethod  # noqa: C901
-    def _get_access_for_user(username, tenant):  # pylint: disable=too-many-locals,too-many-branches
-        """Obtain access data for given username."""
-        principal = None
-        access_list = None
+    def _get_access_for_user(username, tenant):  # pylint: disable=too-many-locals,too-many-branches,unused-argument
```

Comment:
Instead of adding the exception for `unused-argument` here, can we just pull out `username, tenant` here and in the caller? You'd probably have to update the test callers as well.

Reply:
We could, but my worry there was that, since we're attempting to keep the function as is to allow minimal changes to other code, changing its arguments and signature would then require changing any call to the function. I could look into where it's called and fix each call up if there's only a couple, but wanted to go for minimal change where possible.

Reply:
Gotcha, makes sense. I did a global search on the repo and it looks like it's just the one spot calling it (along with tests).